exporter/exporterhelper/README.md

```diff
@@ -24,6 +24,8 @@ The following configuration options can be modified:
       [the batch processor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/batchprocessor)
       is used, the metric `batch_send_size` can be used for estimation)
 - `timeout` (default = 5s): Time to wait per individual attempt to send data to a backend
+- `sampled_logger`: Enables/disables sampling of logging messages, which caps the CPU and I/O load of logging while keeping a representative subset of your logs.
```

Comment:
```suggestion
- `sampled_logger`: Enables/disables sampling of the log messages.
```

we could also replace `Enables/disables` by `Toggles`

Reply:
Keeping `Enables/disables ` for the moment